framework_tvb/tvb/core/services/project_service.py

```diff
@@ -851,6 +852,36 @@ def get_operations_for_datatype(datatype_gid, visibility_filter, only_in_groups=
             return dao.get_operations_for_datatype(datatype_gid, only_relevant=False, only_in_groups=only_in_groups)
         return dao.get_operations_for_datatype(datatype_gid, only_in_groups=only_in_groups)
 
+    def cancel_or_remove_burst(self, burst_id):
```

Comment:
Doesn't it make more sense for these 2 methods to remain in `BurstService` (as they are referring to a BurstConfig instance)?